test/plugins/templates.test.js

```diff
@@ -153,7 +153,7 @@ describe('template plugin test', () => {
         beforeEach(() => {
             options = {
                 method: 'GET',
-                url: '/templates/screwdriver%2Fbuild/1.7.3'
```

Comment:
Yes. But I want to leave it out for now since we haven't really addressed namespace yet.